### ISSUE TITLE
Support disabling rules for single line

### ISSUE THREAD

**Issue by rittneje** [mgechev/revive#648]
**Is your feature request related to a problem? Please describe.**

We want to disable the `error-strings` rule for a single variable. However, revive doesn't seem to have any way to support this.

```go
var (
    // ErrUnauthorized means unauthorized.
    ErrUnauthorized = errors.New("Unauthorized") //revive:disable:error-strings
   
    // ErrFoo should still be flagged by the linter, but isn't.
    ErrFoo = errors.New("Bar")
)
```

Instead, we have to do this, which is pretty awkward and non-standard compared to the `nolint` directive.

```go
var (
    //revive:disable:error-strings

    // ErrUnauthorized means unauthorized.
    ErrUnauthorized = errors.New("Unauthorized")

    //revive:enable:error-strings
   
    // ErrFoo is now flagged by the linter.
    ErrFoo = errors.New("Bar")
)
```

**Describe the solution you'd like**

A way to just disable a rule for a specific line.

**Describe alternatives you've considered**

Since we use golangci-lint, we can do `//nolint:revive`, but this disables all rules, not just `error-strings`.

**Additional context**
n/a

**Comment by chavacava** [comment]
Hi @rittneje, 

Did you try `revive:disable-next-line` or `revive:disable-line` as is documented in the section [Coment Directives](https://github.com/mgechev/revive#comment-directives) at the project's README.md ?

**Comment by rittneje** [comment]
Wow, somehow I missed that sentence. 🤦
